astro/src/content/docs/_shared/_token-storage-options.mdx

```diff
@@ -0,0 +1,15 @@
+| Backend for frontend (BFF)/Sessions | Easy to revoke tokens, works with any client, can be combined with cookie approach to provide cross domain access if some APIs live on different domains                               | Additional server side component, less scalable because you are routing all API requests through the BFF, single point of failure for all API access              | Minimizes token exposure by never sharing tokens with the client, but introduces a new critical infrastructure component that must be secured | <GreenCheck />                                    |
+| Native secure storage   | OS-provided secure storage options like iOS Keychain or Android Keystore prevent token exfiltration       | Platform-specific implementation | Prevents exfiltration by malicious apps, use TLS to prevent MITM attacks                    | <GreenCheck />                                    |
+| Local Storage, IndexedDB         | JavaScript can access token and send requests using `Authorization` or other expected header, supported by some frameworks (Amplify) | Tokens vulnerable to exfiltration by any JavaScript running on the page                                                             | Highly vulnerable to XSS attacks | <RedX /> |
+| In Memory             | Your code can access tokens but other code cannot                                               | Tokens lost when the client reloads                                                                     | Reduces persistence of tokens, limiting exposure | <RedX /> |
+| Web workers           | Isolation of web worker prevents token access | All fetch calls must pass through web worker, have to find library or write web worker code, malicious code may still be able to get data via the web worker, access token lost when page is reloaded | Provides isolation for tokens, but risks persist with malicious code targeting the web worker or intercepting network traffic                | <RedX /> |
+| MTLS                  | Tokens are bound to client, IETF standard                                                      | Not widely supported, requires every client to have an X.509 certificate                                | Offers strong token-binding security, but operational complexity and certificate management introduce significant overhead                    | <RedX /> |
+| DPoP                  | Tokens are bound to client, IETF standard                                                      | Not widely supported, requires APIs to do extra work to verify the token, client must do extra work to send the token, still must take care to avoid exfiltration of key | Enhances token security by binding it to a client, but susceptible to implementation flaws and requires stringent key management              | <RedX /> |
```

Comment:
```suggestion
| Distributed Proof of Possession (DPoP)                  | Tokens are bound to client, IETF standard                                                      | Not widely supported, requires APIs to do extra work to verify the token, client must do extra work to send the token, still must take care to avoid exfiltration of key | Enhances token security by binding it to a client, but susceptible to implementation flaws and requires stringent key management              | <RedX /> |
```